src/lib/viewers/image/ImageViewer.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import get from 'lodash/get';
```

Comment:
We've used `getProp` as a convention since `get` is a reserved key word. See https://github.com/box/box-content-preview/blob/4ca9819f287c36824ef0302f18f87bfcb53234dc/src/lib/viewers/office/OfficeLoader.js#L1 for example